source/2020-08-07-the-ember-times-issue-159.md

```diff
@@ -25,14 +28,18 @@ READMORE
 
 ---
 
-## [Section title in sentence case 🐹](section-url)
+## [How to participate in the Ember RFC process 🚀](https://github.com/emberjs/rfcs/)
 
-<change section title emoji>
 <consider adding some bold to your paragraph>
 <please include link to external article/repo/etc in paragraph / body text, not just header title above>
 
-<add your name to author list, top and bottom>
-<add blurb and emoji to "SOME-INTRO-HERE">
+The Request For Comments (RFC) process is intended to provide a consistent and controlled path for new features to enter the Ember.js framework.
+
+Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow on the appropriate repo. Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the Ember core teams.
```

Comment:
```suggestion
Many changes, including bug fixes and documentation improvements can be implemented and reviewed via the normal GitHub pull request workflow on the appropriate repo. Some changes though are "substantial", and we ask that these be put through a bit of a design process and produce a consensus among the Ember core teams.

So, what's a good way to propose your awesome idea? Well, there are a few options: 

1. Chat with others about your idea in the #dev-rfc channel in Discord. This is a good way to get some initial feedback on your idea from folks in the community who have experience writing RFCs. They can also help you figure out if an RFC is already in the core team's backlog, and what next steps might look like. 
1. Submit an issue to the RFC repo. This is a great way to receive feedback _before_ you start writing the RFC, and will help you maintain efficiency.

Once you are ready to write the RFC for your awesome idea, there are a few important steps: 
1. Read through the entire README so that you understand what your RFC needs to succeed. Sometimes folks will skip over the "How we teach this" section or the "alternatives" section, and both of those are important to include for the RFC to proceed. 
1. Find a core team member to champion the RFC! All RFCs need a core team champion, so review the list of teams to see who you should approach about championing your issue. That core team member makes sure that your RFC is discussed in meetings, so it's essential to have one if you want your awesome idea to succeed! 

If you're interested in eventually writing RFCs but don't know where to start, here are some steps you can take on your own:

* read through the RFCs that have been accepted. Look for the patterns, what similarities do they have?
* read through the issues in the RFC repo. Sometimes folks will know that an RFC needs to be written, but they don't have time to write it. That's a great place to start digging in.
* read through RFC pull requests and read the questions and comments on the PRs. Look for patterns there too- can you anticipate questions and answer them in your RFC before they are asked?

All in all, the RFC process is important to the growth of our community, but it can definitely be a little bit intimidating to know where to start if you're not used to writing RFCs! Hopefully this quick guide gives you a good idea of where to start.
```